src/index.html

```diff
@@ -22,9 +22,10 @@ <h2>Recent Scores</h2>
       <section class="add-score flex">
         <h2 class="add-score">Add your score</h2>
         <form id="scoreForm" class="add-score flex" action="">
-          <input type="text" name="name" placeholder="Your name" />
-          <input type="text" name="score" placeholder="Your score" />
+          <input type="text" name="name" placeholder="Your name" required />
+          <input type="text" name="score" placeholder="Your score" req />
           <input type="submit" value="Submit" />
```

Comment:
- [ Required ] Please consider making the `submit button` work as expected, when the button is clicked it should `hit` the `POST` API endpoint and post user entries. Currently, when I click the button the page is refreshed and no new data is `fetched` from the `API` this shows that the new `entries` were not posted online. Please see the video below for more details on this.

- [ Required ] Please consider making the `refresh button` work as expected, when the button is clicked it should `hit` the `GET` API endpoint and fetch all the scores. Currently, when I click the button the list of scores on the page does not change.

## Video
https://user-images.githubusercontent.com/16392046/217673965-fb4e1958-d444-4232-a6f2-4752d236a02b.mp4